samples/agent-knowledge-hub/python/README.md

```diff
@@ -0,0 +1,179 @@
+---
+page_type: sample
+description: Contoso Knowledge Hub - AI Academic & Career Guidance Agent
+products:
+- office-teams
+- office
+- office-365
+languages:
+- Python
+extensions:
+ contentType: samples
+ createdDate: "10/21/2025 01:38:25 PM"
+urlFragment: officedev-microsoft-teams-samples-agent-knowledge-hub-python
+---
+
+# Contoso Knowledge Hub - AI Academic & Career Guidance Agent
+
+This sample demonstrates an AI-powered Academic Agent built with the **Microsoft Teams AI Library for Python**. The bot provides comprehensive educational support, career guidance, and course recommendations using Azure OpenAI.
```

Comment:
rename from Teams AI Library to Teams SDK

Reply:
maybe best to do CTRL+F and replace cause it's across the files

Reply:
Updated to Teams SDK